src/main/java/pl/touk/sputnik/Test.java

```diff
@@ -5,8 +5,10 @@
     public boolean test = false;
 
     public int method() {
+        String str = null;
         if (test) {
-            return 1;
+          if(test = false)
+            return str.length();
```

Comment:
[FindBugs] WARNING: NP: Null pointer dereference of str in pl.touk.sputnik.Test.method()